fili-core/src/test/groovy/com/yahoo/bard/webservice/async/jobs/payloads/JobPayLoadBuilderSpec.groovy

```diff
@@ -22,16 +24,16 @@ class JobPayLoadBuilderSpec extends Specification {
 
     def "We are able to get the correct results url"() {
         expect:
-        JobPayloadBuilder.getResultsUrl("ticket1", uriInfo) == "https://localhost:9998/v1/jobs/ticket1/results"
+        JobPayloadBuilder.getResultsUrl("ticket1", uriInfo) == "https://localhost:9998/v1/async/ticket1/results"
     }
 
     def "We are able to get the correct syncResults url"() {
         expect:
-        JobPayloadBuilder.getSyncResultsUrl("ticket1", uriInfo) == "https://localhost:9998/v1/jobs/ticket1/results?asyncAfter=never"
+        JobPayloadBuilder.getSyncResultsUrl("ticket1", uriInfo) == "https://localhost:9998/v1/async/ticket1/results?asyncAfter=never"
     }
 
     def "We are able to get the correct self url"() {
         expect:
-        JobPayloadBuilder.getSelfUrl("ticket1", uriInfo) == "https://localhost:9998/v1/jobs/ticket1"
+        JobPayloadBuilder.getSelfUrl("ticket1", uriInfo) == "https://localhost:9998/v1/async/ticket1"
```

Comment:
I checked with Rick and he was under the impression that the api endpoint should still be jobs and not async.  (Applies elsewhere as well)